distutils/sysconfig.py

```diff
@@ -449,6 +449,14 @@ def get_config_vars(*args):
     global _config_vars
     if _config_vars is None:
         _config_vars = sysconfig.get_config_vars().copy()
+        if os.name == 'nt':
+            # See https://github.com/pypa/distutils/issues/130
+            _config_vars['EXT_SUFFIX'] = _imp.extension_suffixes()[0]
+            if not IS_PYPY:
+                # For backward compatibility, see issue19555
+                SO = _config_vars.get('EXT_SUFFIX')
+                if SO is not None:
+                    _config_vars['SO'] = SO
```

Comment:
These lines of code almost double the size of this method, and are presumably only needed for Python 3.9 and earlier. I'll do a refactor to provide a compatibility wrapper with a limited support horizon.

Reply:
Fixed in 74dc5363b3.